tests/pages/services/ServiceActions-cy.js

```diff
@@ -138,7 +128,7 @@ describe('Service Actions', function () {
             response: {
               message: {message: 'Not Authorized to perform this action!'}
             },
-            delay: 50
+            delay: 100
```

Comment:
I think this is fine as it is.
When we talked about it, @jfurrow, I thought you said you used a `cy.wait` which we should try to avoid. I also, can only get this test to pass with the `100` delay.